src/app/docs/lookup/lookup-docs.component.html

```diff
@@ -48,10 +48,78 @@
         headingStyle="success"
       >
         <p class="sky-docs-text-block">
-          Use the lookup component when users know what they are looking for and can select multiple items. For guidance on which selection inputs to use in other scenarios, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design#selection-inputs">form design guidelines</a>.
+          Use the lookup component when users make a single selection or multiple selections from a long list of options. For guidance on which selection inputs to use in other scenarios, see the <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design#selection-inputs">form design guidelines</a>.
         </p>
       </sky-docs-design-guideline>
     </sky-docs-demo-page-section>
+    
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        *skyThemeIf="'default'" imagePath="~/assets/img/guidelines/lookup/lookup-anatomy-default.png"
+      >
+        <sky-docs-anatomy-item>Input field</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Suggestions popover</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Suggested results</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Button to create new option</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+      <sky-docs-anatomy
+        *skyThemeIf="'modern'" imagePath="~/assets/img/guidelines/lookup/lookup-anatomy-modern.png"
+      >
+        <sky-docs-anatomy-item>Input field</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Suggestions popover</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Suggested results</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item isOptional="true">Button to create new option</sky-docs-anatomy-item>
```

Comment:
My understanding was that the terminology guidance to use "New" was specific to modern theme. The button would be labeled "New" in modern theme Lookup, and labeled "Add" in default theme Lookup. Is that incorrect?

Reply:
I'm not sure. The messaging around new development vs. old development and modern theme vs default theme hasn't always been crystal clear. I know it's been communicated that some of the "new" design strategies for new development/modern theme should also be adopted by new development/default theme. For example, Matt Garbarino's vision type calls out that going forward, we change how we use flyouts. That guidance was specific to new development/modern theme, but as I recall, he also said that while we don't need to go back and undo flyouts implemented differently in NXT, we should stop the bleeding and not continue to use flyouts the wrong way when we develop new features for NXT. So I'm not sure how a terminology guideline like this should be treated. We created the Terminology guidelines in the modern theme docs because they are internal, so that gave us a nonpublic forum to post such guidelines, but I don't think we made a clear delineation of whether the guidance applies only to modern theme.

Reply:
Currently it is always "Add". That is potentially a bug though that was missed in review of that feature.

Reply:
Design images are updated to just "New" and my understanding is that the component will likewise be updated, so resolving this conversation...